core/arch/arm/mm/core_mmu.c

```diff
@@ -967,7 +967,9 @@ static void check_pa_matches_va(void *va, paddr_t pa)
 			panic("issue in linear address space");
 		return;
 	}
-	if (v >= (CFG_TEE_LOAD_ADDR & ~CORE_MMU_PGDIR_MASK) &&
+	/* wait pager is initialize before looking into pager data */
+	if (cpu_mmu_enabled() &&
```

Comment:
right , this is not the right test.

i faced this when testing b2260 with pager and playing various sub configs. maybe it's related to `core_mmu_v7.c`. could you test boting tee with pager+debug+info_log on armv8, 64b and 32b modes? i have not checked on the qemus (pager off). i won't be at office until Wednesday, i'll get you the back trace then.